WordPress/WordPressTest/ThemeServiceTests.m

```diff
@@ -165,7 +165,7 @@ - (void)testThatActivateThemeThrowsExceptionWithoutTheme
     ThemeService *service = nil;
 
     blog.dotComID = blogId;
-    blog.account.wordPressComRestApi = api;
+    blog.account._private_wordPressComRestApi = api;
```

Comment:
What do you think about adding the `wordPressComRestApi` setter back to support these statements?

Reply:
It's used only in tests as a way to mock the dependencies, so I think it's OK. It might be worth adding a method like `setTestingWordPressAPI` or something like this. Wdyt? I think this is also acceptable though.

Reply:
Yep. I'm okay with this. The app code should never use the setter after all.